### ISSUE TITLE
php5-mcrypt is installed, but not enabled

### ISSUE THREAD

**Issue by jeffgeorge** [clwdev/precip#5]
For reasons unknown, despite installing `php5-mcrypt` [here](https://github.com/clwdev/precip/blob/master/puppet/precip/manifests/php.pp#L37), it doesn't actually get enabled.

(See output of `$ php -m|grep mcrypt`, it's blank.)

Modules we install should _actually work_.

**Comment by jeffgeorge** [comment]
Terrible Workaround:
`vagrant ssh -c "sudo php5enmod mcrypt && sudo apache2ctl graceful && php -m|grep mcrypt"`

**Comment by jeffgeorge** [comment]
This appears to have been resolved by PR #12:

```
~/www/precip (master)$ vagrant ssh -c "php -m|grep mcrypt"
mcrypt
Connection to 127.0.0.1 closed.
```
